compiler/rustc_hir/src/hir.rs

```diff
@@ -497,7 +497,11 @@ pub struct WhereClause<'hir> {
 
 impl WhereClause<'_> {
     pub fn span(&self) -> Option<Span> {
-        if self.predicates.is_empty() { None } else { Some(self.span) }
+        if self.predicates.is_empty() {
+            None
+        } else {
+            Some(self.span)
+        }
```

Comment:
I suspect this is a result of running `cargo fmt`?
`rustc` uses a pinned unstable version of `rustfmt`, so you have to use `x.py fmt` instead.